java/core/page-factory-utils/src/main/java/ru/sbtqa/tag/pagefactory/util/PageFactoryUtils.java

```diff
@@ -101,7 +101,8 @@ public static Boolean isRequiredAction(Method method, final String title) {
         for (ActionTitle action : actionList) {
             String actionValue = action.value();
             try {
-                I18N i18n = I18N.getI18n(method.getDeclaringClass(), TagCucumber.getFeature().getI18n().getLocale());
+                Scenario currentScenario = ScenarioContext.getScenario();
+                I18N i18n = I18N.getI18n(method.getDeclaringClass(), CucumberUtils.getLocale(currentScenario));
```

Comment:
Scenario currentScenario = ScenarioContext.getScenario();
+                I18N i18n = I18N.getI18n(method.getDeclaringClass(), CucumberUtils.getLocale(currentScenario));

MOVE IT OUT FROM forEach!!!!!